### DISCUSSION TITLE
Upgrading to most current version of Next.js

### DISCUSSION THREAD

**Discussion by antonioOrtiz** [vercel/next.js#12873] [Help]
I am on version `"^9.2.1"` of next in my project:

And I'm also using dependencies:

```
       "next": "^9.2.1",
        "next-authentication": "^0.1.1",
        "next-page-transitions": "^1.0.0-beta.2",
        "next-redux-wrapper": "^3.0.0",
        "next-server": "^9.0.5",
```

So does would upgrading to `9.4` to take advantage of some yummy new features like `New Environment Variables Support` break my app?

Are there docs that explain that explain that?

Also I realized `next-authentication`,  `next-page-transitions`, `next-redux-wrapper` are packages are supporting packages that work with next.js but are not part of core, but what about `next-server`?

BTW I love that discussions are here now. Something felt off with the whole `Spectrum` platform...

**Comment by jamesmosier** [comment]
Unless you are using `next-server` for something I am not aware of, you shouldn't need to explicitly include that as a dependency. `next-server` is built into `next`.

Otherwise, the upgrade from 9.2.1 to 9.4.0 should be seamless. The only way to know for sure is to test it, but I recently performed an upgrade similar to this with no issues. Your other dependencies should be fine as is.
